openlibrary/macros/TypeChanger.html

```diff
@@ -3,6 +3,11 @@
 $# and any generic page in the admin UI; e.g. creating a page.
 <div class="formElement pagetype collapse adminOnly" id="pageType">
     $if ctx.user and ctx.user.is_admin():
+        <style>
```

Comment:
Rather than add an inline style please add this in one of the stylesheets. There is a big effort underway at the moment for us to move away from inline styles and scripts.

To do this we'll need to add a body class in openlibrary/templates/site/body.html or using JavaScript

e.g. 

```
#pageType { display: none;}
.client-admin #pageType {display : block;}
```
We should make sure we don't ever cache the client-admin class for anonymous page views.